modules/importer/src/main/scala/ConfigReader.scala

```diff
@@ -75,10 +76,11 @@ object ConfigReader3 {
       val Instrument    = KeyRead(Tcc.Instrument.Instrument,    Read.instrument)
       val MosPreImaging = KeyRead(Tcc.Instrument.MosPreImaging, Read.yesNo)
       object F2 {
-        val Fpu       = KeyRead(Tcc.Instrument.F2.Fpu,       Read.f2fpu)
-        val Filter    = KeyRead(Tcc.Instrument.F2.Filter,    Read.f2filter)
-        val LyotWheel = KeyRead(Tcc.Instrument.F2.LyotWheel, Read.f2lyotwheel)
-        val Disperser = KeyRead(Tcc.Instrument.F2.Disperser, Read.f2disperser)
+        val Fpu         = KeyRead(Tcc.Instrument.F2.Fpu,         Read.f2fpu)
+        val Filter      = KeyRead(Tcc.Instrument.F2.Filter,      Read.f2filter)
+        val LyotWheel   = KeyRead(Tcc.Instrument.F2.LyotWheel,   Read.f2lyotwheel)
+        val Disperser   = KeyRead(Tcc.Instrument.F2.Disperser,   Read.f2disperser)
+        val WindowCover = KeyRead(Tcc.Instrument.F2.WindowCover, Read.f2windowcover)
```

Comment:
`TCC.Instrument.Filter` is a symbolic representation of the `instrument:filter` key in the sequence config.